src/get-started/flutter-for/react-native-devs.md

```diff
@@ -293,19 +296,22 @@ The `await` operator is used to wait for a `Future`.
 ```dart
 // Dart
 import 'dart:convert';
+
 import 'package:http/http.dart' as http;
 
 class Example {
   Future<String> _getIPAddress() async {
-    final url = new Uri.https('httpbin.org', '/ip');
+    final url = Uri.https('httpbin.org', '/ip');
     final response = await http.get(url);
     String ip = jsonDecode(response.body)['origin'];
     return ip;
   }
 }
 
-main() async {
-  final example = new Example();
+/// Void function could be `async`,
+/// otherwise functions should return `Future`.
```

Comment:
```suggestion
/// An async function returns a `Future`.
/// It can also return `void`, unless you use
/// the `avoid_void_async` lint. In that case,
/// return `Future<void>`.
```